tests/integrationv2/conftest.py

```diff
@@ -23,6 +23,7 @@ def pytest_configure(config):
         set_flag(S2N_NO_PQ, True)
     if fips_mode is 1:
         set_flag(S2N_FIPS_MODE, True)
+        set_flag(S2N_NO_PQ, True)
```

Comment:
Why would we need to do this? Shouldn't we be able to handle fips customers not setting NO_PQ? Otherwise, aren't we potentially breaking existing fips customers?

Reply:
The intention is just to signal that we don't want to run any PQ tests if in FIPS mode. I don't think this breaks anything, since the flags we're setting here are only used internally within integv2 tests. But, I think there's a better way to capture the same functionality by checking the FIPS flag in the S2N provider `supports_cipher` function.